core/templates/dev/head/domain/topic/EditableTopicBackendApiService.js

```diff
@@ -33,7 +33,9 @@ oppia.factory('EditableTopicBackendApiService', [
         });
 
       $http.get(topicDataUrl).then(function(response) {
-        var topic = angular.copy(response.data.topic);
+        var topic = angular.copy(response.data.topic_dict);
```

Comment:
This variable should be called topicDict.

Reply:
Done.